src/patternfly/components/NotificationBadge/examples/notification-badge-simple-example.hbs

```diff
@@ -1,11 +1,11 @@
-{{#> button button--modifier="pf-m-icon" button--attribute='aria-label="Unread notifications"'}}
```

Comment:
oh nice, this should have been `pf-m-plain`. Can you add that class to these buttons?